src/core/User.cpp

```diff
@@ -41,9 +41,10 @@ QList<snowflake> User::guildIDs() const noexcept
     return ret;
 }
 
-QList<QPair<snowflake, QString>> User::pairsGuildIDandHashImg() noexcept
+using std::pair;
+QList<pair<snowflake, QString>> User::pairsGuildIDandHashImg() noexcept
 {
-    QList<QPair<snowflake, QString>> ret;
+    QList<pair<snowflake, QString>> ret;
```

Comment:
Why pair and not QPair?

Reply:
Since Qt6, synonymous with std::pair: [source](https://doc.qt.io/qt-6/qpair-proxy.html)

Reply:
Edit: also synonymous in Qt 5: (source)[https://doc.qt.io/qt-5/qpair.html#details}